runtime/src/lib.rs

```diff
@@ -661,3 +656,44 @@ impl_runtime_apis! {
         }
     }
 }
+
+// The following code cannot be placed out of this crate because of WASM constraints
+
+/// Storage Key for AuthorVrfRandomness
+const AUTHOR_VRF_STORAGE_KEY: [u8; 32] =
+    hex2array!("1cb6f36e027abb2091cfb5110ab5087fd077dfdb8adb10f78f10a5df8742c545");
```

Comment:
Where does this key come from?

Reply:
From the Polkadot UI, this is a storage key for the AuthorVRF, it's not part of the [`well_known_keys`](https://paritytech.github.io/polkadot-sdk/master/cumulus_primitives_core/relay_chain/v8/well_known_keys/index.html) module but follows the same idea